src/Microsoft.VisualStudio.ProjectSystem.Managed/GlobalSuppressions.cs

```diff
@@ -15,3 +15,12 @@
 [assembly: SuppressMessage("Usage", "VSTHRD110:Observe result of async calls", Justification = "https://github.com/dotnet/project-system/issues/7256", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.DependenciesProjectTreeProvider.Initialize")]
 [assembly: SuppressMessage("Usage", "VSTHRD110:Observe result of async calls", Justification = "https://github.com/dotnet/project-system/issues/7256", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Subscriptions.DependenciesSnapshotProvider.SnapshotUpdater.TryUpdate(System.Func{Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Snapshot.DependenciesSnapshot,Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Snapshot.DependenciesSnapshot},System.Threading.CancellationToken)~Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Snapshot.DependenciesSnapshot")]
+
+[assembly: SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.Build.ImplicitlyActiveConfiguredProjectReadyToBuild.GetLatestActivationTask~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.Debug.ILaunchSettingsProvider.WaitForFirstSnapshot(System.Int32)~System.Threading.Tasks.Task{Microsoft.VisualStudio.ProjectSystem.Debug.ILaunchSettings}")]
+[assembly: SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.Input.GetCommandStatusResult.Handled(System.String,Microsoft.VisualStudio.ProjectSystem.CommandStatus)~System.Threading.Tasks.Task{Microsoft.VisualStudio.ProjectSystem.CommandStatusResult}")]
+[assembly: SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.UnconfiguredProjectTasksService.OnProjectFactoryCompleted~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.Threading.Tasks.SequentialTaskExecutor.ExecuteTask(System.Func{System.Threading.Tasks.Task})~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.AbstractMultiLifetimeComponent`1.CreateInitializedInstanceAsync~System.ValueTuple{`0,Microsoft.VisualStudio.Threading.JoinableTask}")]
```

Comment:
This one shouldn't be changed as per the unit test failure:

D:\...\src\Microsoft.VisualStudio.ProjectSystem.Managed\ProjectSystem\AbstractMultiLifetimeComponent.cs(125,64): error VSTHRD200: Avoid "Async" suffix in names of methods that do not return an awaitable type. [D:\workspace\_work\1\s\src\Microsoft.VisualStudio.ProjectSystem.Managed\Microsoft.VisualStudio.ProjectSystem.Managed.csproj]

Reply:
This seems to be a bug. If I remove the suppression, the diagnostic appears in the IDE. If I suppress it again, it generates exactly this same suppression, yet then fails to build.